requirements-dev-sqlalchemy.txt

```diff
@@ -18,5 +18,5 @@ pyodbc>=4.0.30  # sqlalchemy_tests
 snowflake-connector-python==2.3.8  # sqlalchemy_tests
 # END NOTE
 snowflake-sqlalchemy>=1.2.3  # sqlalchemy_tests
-sqlalchemy>=1.3.16 # sqlalchemy_tests
+sqlalchemy>=1.3.16,<1.4.10 # sqlalchemy_tests
```

Comment:
SqlAlchemy 1.4.10 which was released yesterday, appears to cause our MSSQL tests to fail. Temporary pin

Reply:
Will you add a task to remove this pin later to JIRA?

Reply:
done!

Reply:
and will be removing this line since it's already being handled by #2708